src/components/app/invest/AppInvestTokens.tsx

```diff
@@ -288,8 +288,10 @@ export const AppInvestTokens: FC<Props> = ({ className }) => {
 
               // Build underlying symbol
               if (!lTokenSymbol) console.log(data);
+             
+           
               const underlyingSymbol = lTokenSymbol.slice(1);
-
+              console.log(underlyingSymbol);
```

Comment:
Can you remove this console log which seems to be there for debugging purposes only?